src/constraints/generator_constraints.jl

```diff
@@ -89,6 +89,6 @@ function add_gen_constraints(m, p)
         m[:dvRatedProduction][t, ts] for t in p.techs.gen, ts in p.time_steps)
     )
     m[:TotalGenFuelCosts] = @expression(m, p.pwf_e *
-        sum(m[:dvFuelUsage][t,ts] * p.s.generator.fuel_cost_per_gallon for t in p.techs.gen, ts in p.time_steps)
+        sum(m[:dvFuelUsage][t,ts] * p.s.generator.fuel_cost_per_gallon / GAL_DIESEL_TO_KWH for t in p.techs.gen, ts in p.time_steps)
```

Comment:
should this be `* GAL_DIESEL_TO_KWH`? Maybe we should rename the `const` to be more clear how it should be used. Like `GAL_DIESEL_PER_KWH`.

Reply:
Iff you have something in units of A then you can multiply it by A_TO_B to get units of B. If you want to go from B to A, or convert the denominator units from A to B (as is the case here), then you divide by A_TO_B. 
I think all the unit conversion consts used to be A_PER_B actually, but someone changed them to this naming convention. I don't really care either way as long as it's consistent, maybe slightly prefer A_PER_B.